Simplenote/src/main/java/com/automattic/simplenote/models/Preferences.java

```diff
@@ -37,23 +37,29 @@ public boolean getAnalyticsEnabled() {
     }
 
     public ArrayList<String> getRecentSearches() {
-        JSONArray recents = (JSONArray) getProperty(RECENT_SEARCHES_KEY);
+        Object object = getProperty(RECENT_SEARCHES_KEY);
 
-        if (recents == null) {
-            recents = new JSONArray();
-        }
+        if (object instanceof JSONArray) {
+            JSONArray recents = (JSONArray) object;
+
+            if (recents == null) {
```

Comment:
Minor (🔍 ): Consider removing this null check. It seems that this check is no longer necessary since it passes the `if (object instanceof JSONArray) { ... } check, which guarantees that the object can no longer be null. Wdyt?

Reply:
Yep!  That's correct.  I removed the null check in e939966a.